xbmc/platform/posix/filesystem/PosixFile.cpp

```diff
@@ -331,7 +334,39 @@ int CPosixFile::Stat(const CURL& url, struct __stat64* buffer)
   if (filename.empty() || !buffer)
     return -1;
 
+#if defined(HAS_STATX)
+  int dirfd = AT_FDCWD;
+  int flags = AT_SYMLINK_NOFOLLOW;
```

Comment:
This behavior differs from `stat64` - is that intentional?

Reply:
Probably not! I took my best guess based off the documentation I found for statx so if that bit needs changing (resolving symlinks?) then I'll change it.

Reply:
Yes please resolve so behavior matches in the two code paths